src/main/java/org/medibloc/panacea/PanaceaGrpcClient.java

```diff
@@ -43,24 +38,68 @@ public BaseAccount getAccount(String address) throws PanaceaApiException {
 
     public DefaultNodeInfo getNodeInfo() {
         GetNodeInfoRequest request = GetNodeInfoRequest.getDefaultInstance();
-        GetNodeInfoResponse response = tendermintQueryStub.getNodeInfo(request);
+        GetNodeInfoResponse response = grpcStub.getTendermintQueryStub().getNodeInfo(request);
         return response.getDefaultNodeInfo();
     }
 
+    public Topic getTopic(String ownerAddress, String topicName) {
+        QueryGetTopicRequest request = QueryGetTopicRequest.newBuilder()
+                .setTopicName(topicName)
+                .setOwnerAddress(ownerAddress)
+                .build();
+        QueryGetTopicResponse response = grpcStub.getAolQueryStub().topic(request);
+        return response.getTopic();
+    }
+
     public Record getRecord(String ownerAddress, String topicName, Long offset) {
         QueryGetRecordRequest request = QueryGetRecordRequest.newBuilder()
                 .setOwnerAddress(ownerAddress)
                 .setTopicName(topicName)
                 .setOffset(offset)
                 .build();
-        QueryGetRecordResponse response = aolQueryStub.record(request);
+        QueryGetRecordResponse response = grpcStub.getAolQueryStub().record(request);
         return response.getRecord();
     }
 
+    public DIDDocumentWithSeq getDIDDocumentWithSeq(String did) {
+        QueryGetDIDRequest request = QueryGetDIDRequest.newBuilder()
+                .setDID(did)
+                .build();
+        QueryGetDIDResponse response = grpcStub.getDidQueryStub().dIDDocumentWithSeq(request);
+        return response.getDIDDocumentWithSeq();
+    }
+
     public TxResponse broadcast(BroadcastTxRequest request) {
-        BroadcastTxResponse response = txServiceStub.broadcastTx(request);
+        BroadcastTxResponse response = grpcStub.getTxServiceStub().broadcastTx(request);
         return response.getTxResponse();
     }
 
+    public TxResponse getTxResponse(String txHash) {
+        GetTxRequest request = GetTxRequest.newBuilder()
+                .setHash(txHash)
+                .build();
+        GetTxResponse response = grpcStub.getTxServiceStub().getTx(request);
+        return response.getTxResponse();
+    }
+
+    public List<TxResponse> getTxResponsesByHeight(long height) {
```

Comment:
Before spec. Get txResponses required to tx.height.
So I implement it as it is

Reply:
Aha. Yeah. I'm not sure whether SDK users are actually using this features :), but it's good to have because it's simple.